homeassistant/components/bluetooth/__init__.py

```diff
@@ -117,7 +119,7 @@ def manufacturer_id(self) -> int | None:
 
 
 BluetoothChange = Enum("BluetoothChange", "ADVERTISEMENT")
-BluetoothCallback = Callable[[BluetoothServiceInfo, BluetoothChange], None]
+BluetoothCallback = Callable[[BluetoothServiceInfo, BluetoothChange, str], None]
```

Comment:
let's consider either folding the source into `BluetoothServiceInfo` or adding a new data class to hold that data so we can easier expand in the future.

Reply:
Will adjust after https://github.com/home-assistant/core/pull/75195